idl/cherami_server/metadata.thrift

```diff
@@ -401,6 +401,12 @@ service MetadataExposable {
       2: shared.InternalServiceError internalError
     )
 
+  shared.ListConsumerGroupsUUIDResult listConsumerGroupsUUID(1: shared.ListConsumerGroupsUUIDRequest listRequest)
```

Comment:
Could you add some comments about the difference between this one and listConsumerGroups?